src/constants.js

```diff
@@ -0,0 +1,10 @@
+export const Organization = {
+  NAME: 'Recidiviz',
+  DESCRIPTION: 'a platform for tracking granular recidivism metrics in real time',
```

Comment:
Anywhere we say "recidivism metrics" we should generalize to "criminal justice metrics." (Eventually we will want to update all of the licensing, too.)